cafebabel/static/js/articles-tags.js

```diff
@@ -174,12 +169,12 @@ class TagEventListener {
       if (event.keyCode == 40 || event.keyCode == 38) return
```

Comment:
Could you code-comment what these keyCodes are?
Eventually also _why_ to prevent key up and down as it's not obvious from here.